zookeeper/source/zk_dn_client.cc

```diff
@@ -158,8 +158,49 @@ namespace zkclient{
 		if (!zk->create(HEALTH_BACKSLASH + id + BLOCKS, ZKWrapper::EMPTY_VECTOR, error_code)) {
 			LOG(ERROR) << CLASS_NAME <<  "Failed creating /health/<data_node_id>/blocks " << error_code;
 		}
+
+		// Create the work queues, set their watchers
+		std::string q1 = "replicate";
```

Comment:
Can you make this a constant in zk_client_common.h? There's already something called DELETE_QUEUES.